app/blueprints/self_serve/routes.py

```diff
@@ -43,36 +43,6 @@ def index():
     return render_template("index.html")
 
 
-@self_serve_bp.route("/build_form", methods=["GET", "POST"])
-def build_form():
-    form = FormForm()
-    if form.validate_on_submit():
-        new_form = {
-            "builder_display_name": form.builder_display_name.data,
-            "start_page_guidance": form.start_page_guidance.data,
-            "form_display_name": form.form_title.data,
-            "id": human_to_kebab_case(form.form_title.data),
-            "pages": form.selected_pages.data,
-        }
-        save_form(new_form)
-        flash(message=f'Form {new_form["form_display_name"]} was saved')
-        return redirect(url_for("self_serve_bp.index"))
-
-    available_pages = []
-    pages = get_pages_to_display_in_builder()
-    for page in pages:
-        questions = [
-            x["json_snippet"]["title"] if (x := get_component_by_name(comp_name)) else comp_name
-            for comp_name in page["component_names"]
-        ]
-        available_pages.append(
-            {
-                "id": page["id"],
-                "display_name": page["builder_display_name"],
-                "hover_info": {"title": page["form_display_name"], "questions": questions},
-            }
-        )
-    return render_template("build_form.html", available_pages=available_pages, form=form)
 
 
 @self_serve_bp.route("/download_json", methods=["POST"])
```

Comment:
According to the best practices of api naming they say not to use underscores better if we can have it but not forcing to change these :) 

https://restfulapi.net/resource-naming/

Reply:
good catch :) ill make the changes